app/assets/javascripts/qpixel_api.js

```diff
@@ -12,15 +12,40 @@ window.QPixel = {
     return token;
   },
 
+  /** Counts notifications popped up at any time. */
+  let popped_modals_ct = 0;
+
   /**
    * Create a notification popup - not an inbox notification.
    * @param type the type to apply to the popup - warning, danger, etc.
    * @param message the message to show
    */
   createNotification: function(type, message) {
+    // Some messages include a date stamp, `append_date` governs that.
+    let append_date = false;
+    let message_with_date = message;
+    if (type === 'danger') {
+      if (popped_modals_ct > 0 ) {
+        /* At the time of writing, modals stack each one exactly on top of previous one, so repeating an errored action
+         * over and over again is going to create multiple error modals in the same exact place. While this happens this
+         * way, an user closing the error modal will not have an immediate visual action feedback if two or more error
+         * modals have been printed. A date is stamped in order to cope with that. Could be anything. Probablye a cycle
+         * of different emoji characters would be cuter while having the purpose met. But if so make sure character in
```

Comment:
obviously insperd by discord's this thing

<img width="821" alt="Screenshot 2020-08-26 at 02 12 24" src="https://user-images.githubusercontent.com/53513496/91240213-a841da80-e741-11ea-98ce-11355aa6b0f5.png">

<img width="828" alt="Screenshot 2020-08-26 at 02 12 30" src="https://user-images.githubusercontent.com/53513496/91240230-b09a1580-e741-11ea-88b4-6b2e61d62fd1.png">